### PR TITLE
PIM role management tutorial

### PR DESCRIPTION
**Pull request by FaithOmbongi**

### CODE REVIEW

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[a2df1b8](https://github.com/microsoftgraph/microsoft-graph-docs/commits/a2df1b85345b08674fbac1ac5dcc00a20562c1e4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/images/GE-Permissions/User.ReadWrite.All-consent.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/User.ReadWrite.All-consent.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/user.readwrite.all-consent.png?branch=pr-en-us-13764) |
[concepts/images/GE-Permissions/selectpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/selectpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/selectpermissions.png?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/consentpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/consentpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-securitygroup?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/settings.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/settings.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-m365group?branch=pr-en-us-13764) |
[concepts/tutorial-assign-azureadroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/tutorial-assign-azureadroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-assign-azureadroles?branch=pr-en-us-13764) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109032044090679-13764%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=vKe5sh9bHPh3Ys%2B5lHoc%2BzHlgXCQZhT0b1YbZIB9JwM%3D&st=2021-09-03T20%3A40%3A47Z&se=2021-10-04T20%3A45%3A47Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[11fd1c9](https://github.com/microsoftgraph/microsoft-graph-docs/commits/11fd1c98a323ed209705e1d033b51e9ad10d8350)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/images/GE-Permissions/User.ReadWrite.All-consent.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/User.ReadWrite.All-consent.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/user.readwrite.all-consent.png?branch=pr-en-us-13764) |
[concepts/images/GE-Permissions/selectpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/selectpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/selectpermissions.png?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/consentpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/consentpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-securitygroup?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/settings.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/settings.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-m365group?branch=pr-en-us-13764) |
[concepts/tutorial-assign-azureadroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/tutorial-assign-azureadroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-assign-azureadroles?branch=pr-en-us-13764) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109032202541441-13764%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=mt2JGMlomRLCZHz2Zp1Se24IwrCJgL1uaGOcy6c%2BY1Q%3D&st=2021-09-03T21%3A59%3A26Z&se=2021-10-04T22%3A04%3A26Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[6c1f758](https://github.com/microsoftgraph/microsoft-graph-docs/commits/6c1f758ca60fb050d825b73f5e11058f7c4e35ce)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/images/GE-Permissions/User.ReadWrite.All-consent.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/User.ReadWrite.All-consent.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/user.readwrite.all-consent.png?branch=pr-en-us-13764) |
[concepts/images/GE-Permissions/selectpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/selectpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/selectpermissions.png?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/consentpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/consentpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-securitygroup?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/settings.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/settings.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-m365group?branch=pr-en-us-13764) |
[concepts/tutorial-assign-azureadroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/tutorial-assign-azureadroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-assign-azureadroles?branch=pr-en-us-13764) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109032214198099-13764%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=n875TsRRIxJ7e0T7Jn6wxjFAj55fC4YhOrFSdBXYC%2Fw%3D&st=2021-09-03T22%3A10%3A59Z&se=2021-10-04T22%3A15%3A59Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[1119a77](https://github.com/microsoftgraph/microsoft-graph-docs/commits/1119a77e32beb423f25d07ca2cd185d8a89af427)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/images/GE-Permissions/User.ReadWrite.All-consent.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/User.ReadWrite.All-consent.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/user.readwrite.all-consent.png?branch=pr-en-us-13764) |
[concepts/images/GE-Permissions/selectpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/selectpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/selectpermissions.png?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/consentpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/consentpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-securitygroup?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/settings.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/settings.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-m365group?branch=pr-en-us-13764) |
[concepts/tutorial-assign-azureadroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/tutorial-assign-azureadroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-assign-azureadroles?branch=pr-en-us-13764) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109032215040568-13764%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=qjqPCjJ3qPzSHOarHKNw7IL3rICfserRwDjEU057z7s%3D&st=2021-09-03T22%3A14%3A18Z&se=2021-10-04T22%3A19%3A18Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[f95a382](https://github.com/microsoftgraph/microsoft-graph-docs/commits/f95a3821ba2edc41bacdd2143badf115b25d21ea)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/images/GE-Permissions/User.ReadWrite.All-consent.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/User.ReadWrite.All-consent.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/user.readwrite.all-consent.png?branch=pr-en-us-13764) |
[concepts/images/GE-Permissions/selectpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/selectpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/selectpermissions.png?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/consentpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/consentpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-securitygroup?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/settings.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/settings.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-m365group?branch=pr-en-us-13764) |
[concepts/tutorial-assign-azureadroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/tutorial-assign-azureadroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-assign-azureadroles?branch=pr-en-us-13764) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C6%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109061037597057-13764%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=9ig9HilrARmH4Nl7%2FaQYOQhEomQCUnc6b5Vdw46Abks%3D&st=2021-09-06T10%3A34%3A39Z&se=2021-10-07T10%3A39%3A39Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[de8d9c6](https://github.com/microsoftgraph/microsoft-graph-docs/commits/de8d9c6b90beac0a2023e86ad2d31da922e2a4cc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/images/GE-Permissions/User.ReadWrite.All-consent.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/User.ReadWrite.All-consent.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/user.readwrite.all-consent.png?branch=pr-en-us-13764) |
[concepts/images/GE-Permissions/selectpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/selectpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/selectpermissions.png?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/consentpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/consentpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-securitygroup?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/settings.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/settings.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-m365group?branch=pr-en-us-13764) |
[concepts/tutorial-assign-azureadroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/tutorial-assign-azureadroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-assign-azureadroles?branch=pr-en-us-13764) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C6%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109061131485845-13764%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=1jxaten8w%2F4eAgH5S114oqskmj0tRmByUx3iox%2F6RbI%3D&st=2021-09-06T11%3A28%3A52Z&se=2021-10-07T11%3A33%3A52Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[b1b837e](https://github.com/microsoftgraph/microsoft-graph-docs/commits/b1b837e7704bc08914ec6a0ba91ad9865a1f1cc9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/images/GE-Permissions/User.ReadWrite.All-consent.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/User.ReadWrite.All-consent.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/user.readwrite.all-consent.png?branch=pr-en-us-13764) |
[concepts/images/GE-Permissions/selectpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/GE-Permissions/selectpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/images/ge-permissions/selectpermissions.png?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/consentpermissions.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/consentpermissions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-securitygroup?branch=pr-en-us-13764) |
[concepts/images/tutorial-accessreviews-api/settings.png](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/images/tutorial-accessreviews-api/settings.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-accessreviews-m365group?branch=pr-en-us-13764) |
[concepts/tutorial-assign-azureadroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/tutorial-AADRoles/concepts/tutorial-assign-azureadroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/tutorial-assign-azureadroles?branch=pr-en-us-13764) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C6%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109061145369979-13764%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=rhNF6sNHTbfomS0Dh7%2Bb044ylCQX69MMgyoTivSASfg%3D&st=2021-09-06T11%3A42%3A16Z&se=2021-10-07T11%3A47%3A16Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by shauliu1 — APPROVED** [review]

File: concepts/tutorial-assign-azureadroles.md [hunk 1]
```diff
@@ -0,0 +1,500 @@
+---
+title: "Use Microsoft Graph Privileged Identity Management (PIM) API to assign Azure AD privileged roles"
+description: "Use the Privileged Identity Management (PIM) API to assign Azure AD privileged roles"
+author: "FaithOmbongi"
+ms.localizationpriority: medium
+ms.prod: "governance"
+---
+
+# Tutorial: Use the Microsoft Graph Privileged Identity Management (PIM) API to assign privileged roles
```
Comment (Lauragra) [inline concepts/tutorial-assign-azureadroles.md:9]: Please update the title field to match the H1.

Comment (FaithOmbongi) [reply concepts/tutorial-assign-azureadroles.md:9]: @Lauragra Done.

**Comment by Lauragra** [comment]
@FaithOmbongi , I can't verify that the changes are in. Did you do a commit?

**Comment by FaithOmbongi** [comment]
> @FaithOmbongi , I can't verify that the changes are in. Did you do a commit?

Hi @Lauragra , confirmed. Kindly see [this commit](https://github.com/microsoftgraph/microsoft-graph-docs/pull/13764/commits/1c9a0a47f7282eb5d2de97ab7aee28f442093e27).

File: concepts/tutorial-assign-azureadroles.md [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+
+Contoso Limited is a growing service provider that wishes to have its IT Helpdesk manage the lifecycle of employees’ access. They've identified the Azure Active Directory (Azure AD) User Administrator role as the appropriate privileged role required by IT Helpdesk.
```
[the remark this answers is not in the document]
Comment (FaithOmbongi) [reply concepts/tutorial-assign-azureadroles.md:11]: Updated.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**